src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  column-gap: 4px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(n + 5) {
+  background-image: url(../images/star.svg);
+}
```

Comment:
Instead of defining the active and inactive stars separately for each `.stars--` modifier, you can simplify your code by defining the inactive stars as default (as suggested above), and then only changing the active stars for each modifier. For example, you can remove line 17-19 and instead add `.stars--4 .stars__star:nth-child(-n + 4)` selector to line 13-15. Then, you can create similar selectors for `stars--1`, `stars--2`, `stars--3`, and so on, where you only change the number of active stars. This will make your code DRYer and easier to maintain.